src/popup/utils/index.js

```diff
@@ -4,11 +4,16 @@ import { testAccount, txParams } from './config';
 import runMigrations from '../../store/migrations';
 
 export const formatDate = time =>
-  new Date(+time).toLocaleString(navigator.language, {
-    timeStyle: 'short',
-    dateStyle: 'short',
-    hourCycle: 'h23',
-  });
+new Date(+time)
+.toLocaleString(navigator.language, {
+  month: '2-digit',
+  day: '2-digit',
+  year: 'numeric',
+  hour: '2-digit',
+  minute: '2-digit',
```

Comment:
With timeStyle: 'short', dateStyle: 'short' : 
<img width="349" alt="Screenshot 2020-08-14 at 13 58 52" src="https://user-images.githubusercontent.com/7085525/90242866-7aea5800-de36-11ea-80b3-495811827550.png">

With month: '2-digit', day: '2-digit', year: 'numeric', hour: '2-digit', minute: '2-digit':
<img width="348" alt="Screenshot 2020-08-14 at 13 57 28" src="https://user-images.githubusercontent.com/7085525/90242922-93f30900-de36-11ea-9ad0-e4c6adbca4d6.png">

The design:
<img width="445" alt="Screenshot 2020-08-14 at 14 01 34" src="https://user-images.githubusercontent.com/7085525/90242993-c00e8a00-de36-11ea-8372-4a9e8a200ce3.png">

Reply:
Is it really a crucial difference? If we are going to translate the wallet into multiple languages, I think would be more user friendly is to use the default formatting settings. I like `timeStyle`, `dateStyle` options because they allow setting an approximate date length without changing any specific locale settings. I propose to drop even the `hourCycle` property to use a default setting that should be more comfortable for the end-user (considering his locale). The same applies to the discussion above.

Reply:
I have applied the initial initialization of toLocaleString with timeStyle, datesStyle. I agree with your point.
I have set a fixed locale en-US and logged https://github.com/aeternity/superhero-wallet/issues/470. In order to use the current locale we should follow the official standard for locale strings.(Or convert the strings which I think is overcomplication)